packages/stashjs/package.json

```diff
@@ -78,7 +78,7 @@
     "@aws-sdk/client-kms": "^3.38.0",
     "@aws-sdk/client-sts": "^3.34.0",
     "@cfworker/json-schema": "^1.12.1",
-    "@cipherstash/stash-rs": "^0.3.0",
+    "@cipherstash/stash-rs": "workspace:*",
```

Comment:
Does pnpm resolve this to the current version in the workspace exactly? I'm still unfamiliar with pnpm and am curious what actual version gets resolved here.

Reply:
> Does pnpm resolve this to the current version in the workspace exactly?

This is my interpretation of the docs [here](https://pnpm.io/workspaces#publishing-workspace-packages) and [here](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#dependencies).

Reply:
@CDThomas yeah it will always resolve to the current version (rather whatever is in tree). When published it'll use the version of the package.json at the time of publish - I'm not sure if it's more sophisticated than that. This is what we do with `stash-cli` at the moment.

Reply:
Oh @CDThomas I see what you mean - let me check.

Reply:
Yeah looking at the `stash-cli` package it seems to resolve `stashjs` to the exact version using `workspace:*`

Reply:
Thanks! Would `"workspace:^"` be a better option here (for being a little less strict)?

Reply:
I generally like to just hard code versions since packages aren't nice with versions sometimes, but since we're pretty good with semver it is probably fine to use `workspace:^`. We would just need to be strict and make sure we don't do a minor version for anything with an API change.

I'll change it to `workspace:^` 👍🏻

Reply:
Definitely not blocking and I'm not opposed to `"workspace:*"` if you think it's the safer option btw. Mostly curious about the reason behind choosing one over the other here.

Reply:
Considering PRs like this could be avoided in future by using `^` I reckon it makes sense 🤷🏻